nikola/plugins/task/scale_images.py

```diff
@@ -45,6 +45,7 @@ def set_site(self, site):
 
     def process_tree(self, src, dst):
         """Process all images in a src tree and put the (possibly) rescaled images in the dst folder."""
+        thumb_fmt = self.kw['image_thumbnail_format']
         ignore = set(['.svn'])
```

Comment:
`.svn` is the real WTF. And `Thumbs.db`/`.DS_Store` special handling below. Pre-existing WTF, that is.

(You don’t *need* that variable, you could just take it out of `self.kw` on line 65; that would actually look better IMO